instagram/src/components/CommentSection/CommentInput.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const InputForm = styled.form`
+    border: 1px solid white;
+    border-top: 1px solid lightgrey;
+    width: 97%;
+    height: 40px;
+    text-align: left;
+    font-size: 15px;
+    color: grey;
+    margin: 5px 10px;
+`;
+
+const InputDisplay = styled.div`
+
+`;
+
+const CommentInputForm = styled.input`
+    width: 96%;
+    height: 40px;
+    text-align: left;
+    font-size: 15px;
+    color: grey;
+    padding-right: 25px;
+    border: white;
+`;
+
+const Ellipsis = styled.i`
+    color: black;
+`;
+
+const CommentInput = () => {
+    return ( 
+        <div>
+            <InputForm>
+            <InputDisplay>
+                <CommentInputForm type="text" placeholder="Add a comment..." />
+                <Ellipsis className="fas fa-ellipsis-h"></Ellipsis>
```

Comment:
Hm, from what i can tell, `<Ellipsis>` is not needed here since this is where you are adding your comment. This would be useful for the posts that you may be displaying.

Reply:
This is a variable that simply refers to an <i></i> html element that is set up above. It is referring to the ellipsis icon of font awesome. Again, it's styled components, which I think will be taught this coming Wednesday. Thanks!